clang/test/CAS/remote-cache-service.c

```diff
@@ -0,0 +1,38 @@
+// REQUIRES: remote-cache-service, shell
+
+// Need a short path for the unix domain socket.
```

Comment:
We should set this up in in `test/CAS/lit.local.cfg` like we do for `clang-cc1daemon`.

Reply:
Thanks for the pointer! See https://github.com/apple/llvm-project/pull/5355/commits/6128e9f2ae1e573e70bb6bb224e131ce5d08a485